dts/arm/nxp/nxp_k6x.dtsi

```diff
@@ -203,36 +200,27 @@
 			status = "disabled";
 		};
 
+#define K_PIN_DEFINE(k_name, k_port, k_pin, k_mux) \
+	k_name##_##pt##k_port##k_pin: k_name##_##pt##k_port##k_pin { \
+		pin = <k_pin>; \
```

Comment:
Might want to use the properties documented in `Generic pin multiplexing node content` in `pinctrl-bindings.txt`, for consistency.